portfolio/src/test/java/com/google/sps/UserDaoTest.java

```diff
@@ -0,0 +1,32 @@
+// package com.google.sps;
```

Comment:
planning on changing this file in next PR!

Reply:
sounds good